swagger.js

```diff
@@ -32,6 +32,53 @@ const doc = {
         },
       },
     },
+    definitions: {
```

Comment:
You have a `definitions` object here nested under the original definitions object on line 15, which won't be parsed correctly by swagger-autogen.

All objects, regardless of which controller uses them, should be part of the same `definitions` object.

Reply:
Akos, I am sorry, I didn't catch what I should delete as I was working on the code  orders on lines from  35 till 113 , could you please explain? Thank you

Reply:
Currently you have this variable: 

```javascript
const doc = {
  info: {},
  host: "....",
  definitions: {
    PasswordlessUser: {},
    definitions: {
      Order: {},
      NewOrder: {}
    }
  }
}
```

that second "definitions" field shouldn't be there. That's why currently swagger can't find the `Order` definition, because you have it nested under a second `definitions` block.

It should just be:

```
const doc = {
  info: {},
  host: "...",
  definitions: {
    PasswordlessUser: {},
    Order: {},
    OrderList: {},
    ...etc
  }
}
```

Reply:
Thank you , done , commit 58c655b562c8dfe5add91ccfecd671968c917aad